invenio_circulation/transitions/base.py

```diff
@@ -56,13 +56,13 @@ def ensure_required_params(f):
     def inner(self, loan, **kwargs):
         missing = [p for p in self.REQUIRED_PARAMS if p not in kwargs]
         if missing:
-            msg = 'Required input parameters are missing `[{}]`'\
+            msg = "Required input parameters are missing '[{}]'" \
                 .format(missing)
-            raise TransitionConstraintsViolation(msg=msg)
+            raise TransitionConstraintsViolationError(description=msg)
```

Comment:
I think here we should not raise `TransitionConstraintsViolationError` as this exception is used to loop over the available transitions when you trigger a new one. If you have missing a required param this will be valid for every transition. So maybe we need to raise something like `NoRequiredParamsAvailableError`. WDYT?

Reply:
We can raise the existing one `MissingRequiredParameterError`.

Reply:
Yes sure!